lib/libutee/include/tee_api.h

```diff
@@ -114,7 +114,7 @@ bool TEE_MaskCancellation(void);
 TEE_Result TEE_CheckMemoryAccessRights(uint32_t accessFlags, void *buffer,
 				       uint32_t size);
 
-void TEE_SetInstanceData(void *instanceData);
+void TEE_SetInstanceData(const void *instanceData);
```

Comment:
Thanks for pointing out `TEE_PopulateTransientObject()` and `TEE_OpenPersistentObject()`, I just missed them. I'll also update `TEE_Realloc()`.

As for `TEE_GetInstanceData()`, I'm not sure. Adding `const` to the return value will just make it more difficult to callers to use it, no? `[ctx]` means that _"the value is not accessed by the implementation"_, which translates to `const` for parameters, but not for return values I think.

So, I'd like to have:
```
void TEE_SetInstanceData(const void *instanceData);
void *TEE_GetInstanceData(void);
static const void *tee_api_instance_data;
```